### PR TITLE
Show profile and enabled features in version output

### PR DESCRIPTION
**Pull request by zhangmoon**
###  What have you changed?

Show profile and enabled features in version output

###  What is the type of the changes?

- Improvement (a change which is an improvement to an existing feature)

###  How is the PR tested?
- Manual test (add detailed scripts or steps below)

make build:
image

make release:
image

make dist_release:
image

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?
no
###  Does this PR affect `tidb-ansible`?

no

###  Refer to a related PR or issue link (optional)
[https://github.com/tikv/tikv/issues/6195](url)

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -72,12 +72,16 @@ pub fn tikv_version_info() -> String {
...
+         \nRust Version:      {}\
+         \nEnable Features:   {}\
+         \nBuild Profile:     {}",
         env!("CARGO_PKG_VERSION"),
         option_env!("TIKV_BUILD_GIT_HASH").unwrap_or(fallback),
         option_env!("TIKV_BUILD_GIT_BRANCH").unwrap_or(fallback),
         option_env!("TIKV_BUILD_TIME").unwrap_or(fallback),
         option_env!("TIKV_BUILD_RUSTC_VERSION").unwrap_or(fallback),
+        option_env!("TIKV_ENABLE_FEATURES").unwrap_or(fallback),
+        option_env!("TIKV_BUILD_PROFILE").unwrap_or(fallback),
```
Comment (nrc) [inline src/lib.rs]: I think using `PROFILE` is fine for the build profile.

**Comment by zhangmoon** [comment]
It is my first time to create pull request. 
WeChatc137e2ce3a2cc7fcd219dae26d7ddef4。
what should I do with this error?

**Comment by nrc** [comment]
> what should I do with this error?

```
$ git commit -s --amend`
$ git push -f
```

**Comment by zhangmoon** [comment]
replaced with https://github.com/tikv/tikv/pull/6215

**Pull request closed without merging**